sdk/python/feast/feature_store.py

```diff
@@ -768,11 +779,17 @@ def get_online_features(
             >>> online_response_dict = online_response.to_dict()
         """
         _feature_refs = self._get_features(features, feature_refs)
-        all_feature_views = self._list_feature_views(
-            allow_cache=True, hide_dummy_entity=False
+        all_feature_views = self._registry.list_feature_views(
+            project=self.project, allow_cache=True
+        )
```

Comment:
I think we decided to move to using FeatureStore versions of `list_feature_views`. In particular, here we wanted to include the info of the dummy entity in this internal version, but hide it from the user-facing functions

Reply:
thanks, this was useful.